front-end-notes/psuedo-selectors.md

```diff
@@ -0,0 +1,24 @@
+#Psuedo-class selectors
+
+###A CSS pseudo-class is a keyword added to selectors that specifices a speical state of the element to be selected.
+
+##### Example
+```css
+ selector: pseduo-class {
+  property: value;
+  };

```

Comment:
you don't need a semi-colon after the closing bracket on line 9.. what is this .. JSCSS? lol